include/SFML/Graphics/Rect.inl

```diff
@@ -118,36 +110,19 @@ constexpr std::optional<Rect<T>> Rect<T>::findIntersection(const Rect<T>& rectan
 }
 
 
-////////////////////////////////////////////////////////////
-template <typename T>
-constexpr Vector2<T> Rect<T>::getPosition() const
-{
-    return Vector2<T>(left, top);
-}
-
-
-////////////////////////////////////////////////////////////
-template <typename T>
-constexpr Vector2<T> Rect<T>::getSize() const
-{
-    return Vector2<T>(width, height);
-}
-
-
 ////////////////////////////////////////////////////////////
 template <typename T>
 constexpr Vector2<T> Rect<T>::getCenter() const
 {
-    return getPosition() + getSize() / T{2};
+    return position + size / T{2};
 }
 
 
 ////////////////////////////////////////////////////////////
 template <typename T>
 constexpr bool operator==(const Rect<T>& left, const Rect<T>& right)
 {
-    return (left.left == right.left) && (left.width == right.width) && (left.top == right.top) &&
-           (left.height == right.height);
+    return (left.position == right.position) && (left.size == right.size);
```

Comment:
Might be a good opportunity to change `left` and `right` into `lhs` and `rhs` respectively, as it can be confusing to read `left` in this context.

Reply:
Done. Is that a convention we would everywhere else?